src/app/shared/custom-components/offer-item/offer-item.component.scss

```diff
@@ -1,135 +1,157 @@
 @import '/src/app/styles/variables.scss';
-
-:host::ng-deep .mat-card-header-text {
-  margin: 0.6em;
-}
+$small-card: 1500px;
```

Comment:
You might leave a comment on what is it or even better put this into our special file with CSS variables

Reply:
this is the width for the card only, I doubt that we will need this value in other places, so I defined it only in this component. I can leave a comment